tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3277,4 +3283,38 @@ public function userFavoritesElementInSpaceUsingTheWebdavApi(string $user, strin
 		$this->setSpaceIDByName($user, $spaceName);
 		$this->favoritesContext->userFavoritesElement($user, $path);
 	}
+
+	/**
+	 * @Given /^user "([^"]*)" has stored id of (?:file|folder) "([^"]*)" of the space "([^"]*)"$/
```

Comment:
```suggestion
	 * @Given /^user "([^"]*)" has stored id of (file|folder) "([^"]*)" of the space "([^"]*)"$/
```
may be this? the down one is like this